pages/docs/data-ops/cli/v3/pull.mdx

```diff
@@ -46,4 +46,44 @@ For example: if you export a resource, it will default to being exported with a
 
 ## dbt models and the `pull` command
 
-Currently, the Hashboard CLI does not automatically update Hashboard metadata inside of dbt files, so any dbt-based models are skipped when evaluating the `pull` command.
+You can pull Hashboard model definitions directly into your dbt files using the `--include-dbt` flag.
+
+```bash
+# pull all resources controlled by code including dbt models
+hb pull --include-dbt
+
+# pull a single dbt model with either an alias or id
+hb pull m:s3AdinvBhsdJ --include-dbt
+hb pull m::my_sales_model --include-dbt
+```
+
+This will update the Hashboard metadata directly in your dbt properties yml files.
```

Comment:
Maybe mention somewhere that if you exclude this flag in `hb pull`, dbt-linked models will be skipped?